news/_posts/releases/2020-10-22-prestashop-1-7-7-0-rc-release.md

```diff
@@ -0,0 +1,108 @@
+---
+layout: post
+title:  "PrestaShop 1.7.7.0 RC1 Is Ready To Be Tested!"
+subtitle: "A month after the beta, here it is: the first release candidate for PrestaShop 1.7.7.0 is now ready for you to test!"
+date:   2020-11-03 15:00:00
```

Comment:
Does the file need to be renamed? I believe it is used for the sorting no? and maybe the url?

Reply:
Yes better rename it